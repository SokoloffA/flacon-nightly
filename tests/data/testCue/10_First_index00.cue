PERFORMER "Artist"
TITLE "Album"
REM GENRE Heavy Metal
REM DATE 2001
FILE "Artist - Album, 2001 (LP1).flac" WAVE
  TRACK 01 AUDIO
    TITLE "Track 1.1"
    INDEX 01 00:03:00
  TRACK 02 AUDIO
    TITLE "Track 1.2"
    INDEX 01 04:38:00


FILE "Artist - Album, 2001 (LP2).flac" WAVE
  TRACK 09 AUDIO
    TITLE "Track 2.1"
    INDEX 01 00:10:00
  TRACK 10 AUDIO
    TITLE "Track 2.2"
    INDEX 01 05:24:00

