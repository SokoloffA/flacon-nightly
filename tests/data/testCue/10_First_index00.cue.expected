[DISC 01 / TRACK 01]
	 FILE        = Artist - Album, 2001 (LP1).flac
	 INDEX CD 00 = 00:00:00
	 INDEX CD 01 = 00:03:00

[DISC 01 / TRACK 02]
	 FILE        = Artist - Album, 2001 (LP1).flac
	 INDEX CD 00 = 04:38:00
	 INDEX CD 01 = 04:38:00




[DISC 01 / TRACK 03]
	 FILE        = Artist - Album, 2001 (LP1).flac
	 INDEX CD 00 = 00:00:00
	 INDEX CD 01 = 00:10:00

[DISC 01 / TRACK 04]
	 FILE        = Artist - Album, 2001 (LP1).flac
	 INDEX CD 00 = 05:24:00
	 INDEX CD 01 = 05:24:00

